src/elements/hx-accordion-panel/_shadow.scss

```diff
@@ -2,6 +2,7 @@
 
 #hxToggle {
   background-color: transparent;
+  outline: 0;
```

Comment:
Good effort, though we still need to show focus when a user tabs through the Stepper. 

Please make the focus similar to http://helix.rax.io/style/focus-states.html

Reply:
Changes have been done, please review.